pytorch/inference/docker/1.5.1/py38/Dockerfile.eia

```diff
@@ -0,0 +1,137 @@
+FROM ubuntu:20.04
+LABEL maintainer="Amazon AI"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL dlc_major_version="2"
```

Comment:
It looks like a new python version constitutes as a new version stream. As a result, we can change this back to dlc_major_version="1"

Reply:
Modified it back to "1"